src/server/pfs/s3/driver.go

```diff
@@ -50,19 +50,28 @@ func NewMasterDriver() *MasterDriver {
 }
 
 func (d *MasterDriver) listBuckets(pc *client.APIClient, r *http.Request, buckets *[]*s2.Bucket) error {
-	repos, err := pc.ListRepo()
+	repos, err := pc.ListRepoByType("") // get repos of all types
 	if err != nil {
 		return err
 	}
 
 	for _, repo := range repos {
+		if repo.Repo.Type == pfs.SpecRepoType {
```

Comment:
My understanding of the code was that leaving a bucket out of the response here doesn't actually prevent it from being accessed directly by name, but maybe that's incorrect. Also not sure on whether to include stats/meta repos by default